postgres/candy_friends.py

```diff
@@ -0,0 +1,21 @@
+
+def show_favorite_candy():
+    import psycopg2
+    try:
+        conn = psycopg2.connect("dbname='adhoc_db' user='postgres' host='localhost' password='RustyLemon'")
```

Comment:
`password='RustyLemon'"` The whole world knows this password now.

If you have time before we meet see if you can figure out how to use https://pypi.org/project/python-dotenv/